torch/csrc/jit/symbolic_script.cpp

```diff
@@ -708,16 +715,32 @@ const std::vector<std::string> functions = {
                 output = bn_out
 
             def backward(grad_output):
-                if weight is not None:
-                    grad_output = grad_output * torch.t(weight)
-                    weight = grad_output * torch.t(bn_out)
+                if weight is not None and bias is not None:
+                    grad_bn_out = (grad_output * weight)._grad_sum_to_size(bn_out.size())
+                    grad_weight = (grad_output * weight)._grad_sum_to_size(weight.size())
+                    grad_bias = grad_output._grad_sum_to_size(bias.size())
+                elif weight is not None:
+                    grad_bn_out = grad_output._grad_sum_to_size(bn_out.size())
+                    grad_weight = (grad_output * bn_out)._grad_sum_to_size(weight.size())
+                    grad_bias = None
+                elif bias is not None:
+                    grad_bn_out = grad_output._grad_sum_to_size(bn_out.size())
```

Comment:
_grad_sum_to_size is not required, same for the `grad_bn_out` in the last conditional